pkg/core/region_tree.go

```diff
@@ -342,3 +346,20 @@ func (t *regionTree) TotalWriteRate() (bytesRate, keysRate float64) {
 	}
 	return t.totalWriteBytesRate, t.totalWriteKeysRate
 }
+
+func (t *regionTree) AtomicAddStaleRegionCnt() {
+	if t.length() == 0 {
+		return
+	}
+	atomic.AddInt64(&t.staleRegionCnt, 1)
+}
+
+func (t *regionTree) AtomicSubStaleRegionCnt() {
+	if t.length() == 0 {
+		return
+	}
+	if t.staleRegionCnt == 0 {
```

Comment:
Do we need to do `atomic.LoadInt64` here?

Reply:
fixed